### ISSUE TITLE
Missing top-level class documentation comment.

### ISSUE THREAD

**Issue by asiniy** [rubocop/rubocop#691]
I'm new to rubocop, and one function is annoying to me.

Why I get `Missing top-level class documentation comment.` before each class or module definition?

``` ruby
class UsersController < InheritedResources::Base
^^^^^
```

**Comment by justinschier** [comment]
Can I ask what WOULD be the correct way to write my class definition so that it passes the rubocop check?

```
class LocationFeatureSchedule < ActiveRecord::BasePIOPS2
  belongs_to :location
end
```

**Comment by jonas054** [comment]
@justinschier You write a comment above it (describing what it is).

**Comment by Willibaur** [comment]
Thanks @bbatsov  always good to learn best practices

**Comment by rohan2609** [comment]
how to solve offences in rubocp

**Comment by giovannibenussi** [comment]
Using this doesn't work for me:
```yml
AllCops:
  Documentation:
    Enabled: false
```

However, removing `AllCops` does the trick:

```yml
Documentation:
  Enabled: false
```

**Comment by nghiabk** [comment]
I have the same problem. I have fixed 

# Your comment
class UsersController < InheritedResources::Base

**Comment by wildaspt** [comment]
@nghiabk change #Your comment into # Your comment
that will solve your problem

**Comment by lanzhiheng** [comment]
I think we should use 

```
Style/Documentation:
  Enabled: false
```

to avoid  the document checking.

![image](https://user-images.githubusercontent.com/8858472/88059762-df97f700-cb97-11ea-92bc-e6ca1485d077.png)

 otherwise, without the typename, I will get this warning(I am working in version 0.88)

```
/xxxxxx/.rubocop.yml: Warning: no department given for Documentation.
```

**Comment by Maxeeezy** [comment]
This is the right one, thank you @lanzhiheng !
